src/Discord.Net.Commands/Attributes/DescriptionAttribute.cs

```diff
@@ -12,7 +13,8 @@ public DescriptionAttribute(string text)
         }
     }
 
-    [AttributeUsage(AttributeTargets.Method)]
+    // Brief summary of method/module
```

Comment:
Synopsis is usually defined as 'a brief summary' (in contrast to description, which is defined as 'a spoken or written account of a person, object, or event' - basically a 'full summary') which is why I changed the meanings about.